features/git-rename-branch/non_feature_branch/with_remote_origin/with_tracking_branch/with_unpulled_changes.feature

```diff
@@ -0,0 +1,34 @@
+Feature: git rename-branch: errors if renaming a non-feature branch that has unpulled changes
+
+  As a developer renaming a non-feature branch that has unpulled changes
+  I should get an error that the given branch is not in sync with its tracking branch
+  So that I don't lose work by accidentally overwriting existing branches.
```

Comment:
Let's use the same line here as in the corresponding feature spec for renaming feature branches.